dell-csi-helm-installer/verify.sh

```diff
@@ -210,6 +210,7 @@ function verify_k8s_versions() {
   if [[ ${V} > ${MAX} ]]; then
     error=1
     found_warning "Kubernetes version ${V} is newer than the version that has been tested. Latest tested version is: ${MAX}"
+    found_warning "To ensure the driver is fully supported run cert-csi and make sure all tests pass. More details: https://dell.github.io/csm-docs/docs/cert-csi/"
```

Comment:
Can we add in the message stating that "It is not a qualified version in the beginning"

Reply:
This is basically what above sentence says and that message has been approved so...
I'm not changing it ; hope that fine :-)